### PR TITLE
Update macos.md

### PR DESCRIPTION
**Pull request by chrbeckm**

### CODE REVIEW

File: install/macos.md [hunk 1]
```diff
@@ -76,9 +76,18 @@ Diese startest du mit
...
+
+im Terminal ein. In der ausgegebenen Liste sollte `Mambaforge-Darwin-arm64.sh`, oder ähnlich, auftauchen.
+Wenn das so ist, kannst du sie mit dem folgenden Befehl löschen
+
+    rm Mambaforge-Darwin-arm64.sh
+    
+oder dem entsprechenden Dateinamen.
+Falls du dir nicht sicher bist, kannst du uns für diesen Punkt auch im Workshop ansprechen.
 
-    rm ~/.local/Mambaforge-*.sh
```
Comment (jpherdi) [inline install/macos.md:81]: this was plain wrong, right?
How should this have moved in .local?
Just following the manual one would still be in the home folder I think

Comment (jpherdi) [reply install/macos.md:81]: Do we need all these extra lines, or would it be enough to just fix it? @chrbeckm

**Comment by jpherdi** [comment]
We should fix Max' second comment too.
code -> codium

File: install/macos.md (hunk 1 above)
Comment (chrbeckm) [reply install/macos.md:81]: Those are the lines from the linux/w10 notrs, so yes, the Mambaforge.sh shouldn't be in .local.

**Review by jpherdi — APPROVED** [review]

**Pull request merged**